comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java

```diff
@@ -141,8 +141,6 @@ public WsSaleRef closeSale(@PathParam("id") long id) {
     @Path("{id}/payed")
     public BigDecimal getSaleTotalPayed(@PathParam("id") long id) {
         Sale sale = saleService.findSaleById(id);
-        saleStateChecker.checkState(sale, false); // TODO: replace with bean validation
-
         return saleService.getSaleTotalPayed(sale);
```

Comment:
Why, closed sales are totally payed anyway? Do you display payments for
closed sales somewhere?

Le mer. 26 août 2015 à 13:12, cghislai <notifications@github.com> a écrit :

> In
> comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java
> <https://github.com/Valuya/comptoir/pull/56#discussion_r37970162>:
>
> > @@ -141,8 +141,6 @@ public WsSaleRef closeSale(@PathParam("id") long id) {
> >      @Path("{id}/payed")
> >      public BigDecimal getSaleTotalPayed(@PathParam("id") long id) {
> >          Sale sale = saleService.findSaleById(id);
> > -        saleStateChecker.checkState(sale, false); // TODO: replace with bean validation
> > -
> >          return saleService.getSaleTotalPayed(sale);
>
> For simplicity, I call this as well when displaying closed sales
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Valuya/comptoir/pull/56/files#r37970162>.
>